src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -1,73 +1,109 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import moment from 'moment';
+
 import { focusElement } from '../../../../platform/utilities/ui';
 import OMBInfo from '@department-of-veterans-affairs/jean-pants/OMBInfo';
 import FormTitle from '../../../common/schemaform/components/FormTitle';
+import { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import { toggleLoginModal } from '../../../login/actions';
 
-import SaveInProgressIntro, { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import formConfig from '../config/form';
+import { submitIntentToFile } from '../actions';
+import FormStartControls from './FormStartControls';
 
 class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  hasSavedForm = () => {
+    const { saveInProgress: { user } } = this.props;
+    return user.profile && user.profile.savedForms
+      .filter(f => moment.unix(f.metadata.expires_at).isAfter())
+      .find(f => f.form === this.props.formId);
+  }
+
+  updateITFStatus = (ITFStatus) => {
+    this.setState(ITFStatus);
```

Comment:
It's not immediately obvious why this works; I had to hunt around. Can we change the `submitIntentToFile` action to return a string so this isn't affected by an arbitrary variable rename in a separate file? That would be...hard to debug.

Reply:
Sounds good, updated.